src/clr-angular/utils/conditional/if-open.service.spec.ts

```diff
@@ -56,6 +61,23 @@ export default function(): void {
                 test = ifOpenService.open;
                 expect(test).toEqual(init);
             });
+
+            it("provides a method to register elements on which click events are ignored while toggling popovers",
+               () => {
+                   expect(ifOpenService.registerIgnoredElement).toBeDefined();
+               });
+
+            it("provides an observable for ignored element changes", () => {
+                let testEl: ElementRef;
+                const sub: Subscription = ifOpenService.ignoredElementChange.subscribe((el: ElementRef) => {
+                    testEl = el;
+                });
+
+                ifOpenService.registerIgnoredElement(new ElementRef(null));
+
+                expect(testEl).not.toBeUndefined();
```

Comment:
Nitpick: wouldn't it be slightly better to do this, since it's pretty much free?
```typescript
const el = new ElementRef(null);
ifOpenService.registerIgnoredElement(el);
expect(testEl).toBe(el);
```

Reply:
done.